styles/style.css

```diff
@@ -0,0 +1,75 @@
+.container {
+   
+    width: 1900px;
+    height: 900px;
+    display: grid; 
+    grid-template-columns: 1fr 1fr ;
+    grid-template-rows: 1fr 1fr 1fr ;
+    
+}
+
+.flex-container {
+   
+    width: 1900x;
+    height: 900px;
+    display: flex; 
+    flex-wrap:nowrap;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    
+}
+ h1{
+
+    background-color: pink;
+    grid-column: span 5;
+    grid-row: 1;
+    padding: 1em;
+    text-align: center;
+    font-size: 30pt;
+    font-family: sans-serif;
+    color:rgb(47, 60, 107);
+    border: white 5px solid;
+    border-style: dashed;
+
+}
+
+h2 {
+    text-align: center;
+    font-family:cursive;
+    font-size: larger;
+    
+}
+
+nav {
+   
+    background-color: pink;
+    grid-column: 1/5;
```

Comment:
Not all of your nav tags are in grids. Try to make your rules more specific. And for the one that is in a grid, there are only 2 columns, but your end column tries to span 4 columns.